src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    double getFigureArea();
```

Comment:
Better to rename this method on `calculateFigureArea` to avoid misunderstanding that it is a getter for field